src/containers/shared/utils.js

```diff
@@ -438,3 +438,41 @@ export const fetchMetrics = () =>
 
 export const removeRoutes = (routes, ...routesToRemove) =>
   routes.filter((route) => !routesToRemove.includes(route.title))
+
+export const formatAsset = (asset) =>
+  typeof asset === 'string'
+    ? { currency: 'XRP' }
+    : {
+        currency: asset.currency,
+        issuer: asset.issuer,
+      }
+
+export const formatBalance = (asset) => {
```

Comment:
This looks to be the same a `formatAmount in` `src/rippled/lib/txSummary/formatAmount.js`

Reply:
removed

Reply:
Doesn't look like this was removed

Reply:
bump

Reply:
Lets remove this

Reply:
weird, removed again